2008-11-07  Arnaud Charlet  <charlet@adacore.com>

	* configure: Disable dolt, which is even worse than libtool for us
	(using too recent bash syntax).

2008-11-05  Arnaud Charlet  <charlet@adacore.com>

	* src/cairo-xlib-xrender-private.h: Fix build on systems that do
	not have Render 0.10.
	Fixed in cairo 1.8.4, see
        https://bugs.freedesktop.org/show_bug.cgi?id=18384

2008-10-30  Arnaud Charlet  <charlet@adacore.com>

	* src/cairo-quartz-font.c: Add missing prototype

--- src/cairo-quartz-font.c.old	2008-10-30 11:36:44.000000000 +0100
+++ src/cairo-quartz-font.c	2008-10-30 12:35:26.000000000 +0100
@@ -786,6 +786,8 @@ _cairo_quartz_scaled_font_get_cg_font_re
 cairo_font_face_t *
 cairo_quartz_font_face_create_for_atsu_font_id (ATSUFontID font_id)
 {
+    extern ATSFontRef FMGetATSFontRefFromFont (FMFont iFont);
+
     ATSFontRef atsFont = FMGetATSFontRefFromFont (font_id);
     CGFontRef cgFont = CGFontCreateWithPlatformFont (&atsFont);
     cairo_font_face_t *ff;
--- src/cairo-xlib-xrender-private.h.old	Wed Nov  5 19:08:50 2008
+++ src/cairo-xlib-xrender-private.h	Wed Nov  5 19:09:06 2008
@@ -45,6 +45,23 @@
 #include <X11/extensions/Xrender.h>
 #include <X11/extensions/renderproto.h>
 
+/* We require Render >= 0.6.  The following defines were only added in
+ * 0.10.  Make sure they are defined.
+ */
+
+/* Filters included in 0.10 */
+#ifndef FilterConvolution
+#define FilterConvolution                   "convolution"
+#endif
+
+/* Extended repeat attributes included in 0.10 */
+#ifndef RepeatNone
+#define RepeatNone                          0
+#define RepeatNormal                        1
+#define RepeatPad                           2
+#define RepeatReflect                       3
+#endif
+
 #else /* !CAIRO_HAS_XLIB_XRENDER_SURFACE */
 
 /* Provide dummy symbols and macros to get it compile and take the fallback
--- configure.old	2008-11-07 17:00:33.093987080 +0100
+++ configure	2008-11-07 17:03:34.222451368 +0100
@@ -20404,5 +20404,5 @@ if test x$DOLT_BASH = x; then
 fi
-if test x$GCC != xyes; then
+#if test x$GCC != xyes; then
     dolt_supported=no
-fi
+#fi
 case $host in
